interface/LeptonEfficiencySF.h

```diff
@@ -863,22 +887,28 @@ class LeptonEfficiencySF
         }else if(id==169){
             if (is2016){
                 if(etaCen<0.9){
-                    if(etaFwd<0.9){ eff.first=0.9490; eff.second=0.001;
-                    }else if(etaFwd<1.2){ eff.first=0.9628; eff.second=0.005;
-                    }else if(etaFwd<2.1){ eff.first=0.9477; eff.second=0.005;
-                    }else if(etaFwd<2.4){ eff.first=0.9420; eff.second=0.010;
+                    if(etaFwd<0.9){ eff.first=0.9956; eff.second=0.0865;
+                    }else if(etaFwd<1.2){ eff.first=0.9930; eff.second=0.0612;
+                    }else if(etaFwd<2.1){ eff.first=0.9964; eff.second=0.0612;
+                    }else if(etaFwd<2.4){ eff.first=0.9920; eff.second=0.0616;
                     }
                 }else if(etaCen<1.2){
-                    if(etaFwd<1.2){ eff.first=0.9576; eff.second=0.005;
-                    }else if(etaFwd<2.1){ eff.first=0.9493; eff.second=0.010;
-                    }else if(etaFwd<2.4){ eff.first=0.9142; eff.second=0.015;          
+                    if(etaFwd<0.9){ eff.first=0.9930; eff.second=0.0612;
+                    }else if(etaFwd<1.2){ eff.first=0.9902; eff.second=0.0057;
+                    }else if(etaFwd<2.1){ eff.first=0.9940; eff.second=0.0046;          
+                    }else if(etaFwd<2.4){ eff.first=0.9875; eff.second=0.0075;          
                     }
                 }else if(etaCen<2.1){
-                    if(etaFwd<2.1){ eff.first=0.9216; eff.second=0.010;
-                    }else if(etaFwd<2.4){ eff.first=0.9470; eff.second=0.015;
+                    if(etaFwd<0.9){ eff.first=0.9964; eff.second=0.0612;
+                    }else if(etaFwd<1.2){ eff.first=0.9940; eff.second=0.0046;
+                    }else if(etaFwd<2.1){ eff.first=0.9972; eff.second=0.0030;          
+                    }else if(etaFwd<2.4){ eff.first=0.9935; eff.second=0.0065;          
                     }
                 }else if(etaCen<2.4){
-                    if(etaFwd<2.4){ eff.first=0.9252; eff.second=0.025;
+                    if(etaFwd<0.9){ eff.first=0.9920; eff.second=0.0615;
+                    }else if(etaFwd<1.2){ eff.first=0.9875; eff.second=0.0074;
+                    }else if(etaFwd<2.1){ eff.first=0.9935; eff.second=0.0065;          
+                    }else if(etaFwd<2.4){ eff.first=0.9855; eff.second=0.0088;          
```

Comment:
Here also you should keep the old way of doing: etaCent < etaFwd by construction. Also as you see the SF that you computed are symmetrical. (in other words in the else if(etaCen<2.4){, the (etaFwd<0.9){ and so have no sense !)